lib/routes/account.js

```diff
@@ -390,6 +395,26 @@ module.exports = function (
             )
         }
 
+        function checkNumberOfActiveSessions () {
+          return db.sessions(emailRecord.uid)
```

Comment:
If this whole function is *solely* for metrics purposes, would we rather this occur in a `setTimeout`? This PR adds a db call for a user's sessions, only to log a warning, and the response must wait for all that before confirm a login.